Tasks/PowerShellV2/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -16,6 +16,8 @@
   "loc.input.help.errorActionPreference": "Prepends the line `$ErrorActionPreference = 'VALUE'` at the top of your script.",
   "loc.input.label.failOnStderr": "Fail on Standard Error",
   "loc.input.help.failOnStderr": "If this is true, this task will fail if any errors are written to the error pipeline, or if any data is written to the Standard Error stream. Otherwise the task will rely on the exit code to determine failure.",
+  "loc.input.label.showWarnings": "Show warnings as Azure DevOps warnings",
```

Comment:
Is this changes related to migration?

Reply:
no, it looks like in [this pr](https://github.com/microsoft/azure-pipelines-tasks/pull/13122) changes to task.loc weren't included, but they should have been

Reply:
Let's update it also here since they haven't been included previously - @DaniilShmelev could you please mention it in PR description also?

Reply:
done!